src/components/FormBody/FormBody.jsx

```diff
@@ -13,10 +13,24 @@ export const FormBody = ({
 
   let formBodyClassnames = 'form'
 
+  /**
+   * Prevents form submission when Enter is pressed on any form element
+   * @param {Object} e event object
+   */
+  const onKeyDown = (keyPressEvent) => {
+    if (keyPressEvent.key === 'Enter') {
+      keyPressEvent.preventDefault()
+    }
+  }
+
   formBodyClassnames += ` ${className}`
 
   return (
-    <div className={elementClasses(formBodyClassnames, 'card')}>
+    // eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions, jsx-a11y/no-static-element-interactions
```

Comment:
I think you only need `jsx-a11y/no-static-element-interactions` here, not both.

We really don't want to ever disable an eslint rule, but in this case you do need to disable it. This use case is actually called out in the [rule documentation](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/a7d1a12a6198d546c4a06477b385b4fde03b762e/docs/rules/no-static-element-interactions.md#case-the-event-handler-is-only-being-used-to-capture-bubbled-events) for being a valid reason to disable the rule

When something is disabled, add a comment explaining why it is necessary. Something like "Disabling because we want to capture events from bubbling up" is enough